src/NovelRT/Graphics/RenderingService.cpp

```diff
@@ -15,7 +15,8 @@ namespace NovelRT::Graphics {
       glBindBufferRange(GL_UNIFORM_BUFFER, 0, tempHandle, 0, sizeof(Maths::GeoMatrix4x4<float>));
       return tempHandle;
     })),
-    _camera(nullptr) {
+    _camera(nullptr),
+    _framebufferColour(RGBAConfig(0,0,0,0)) {
```

Comment:
This is where our default colour is determined so that we always have _something_ passing into our `glClearColor` call.

Reply:
Shouldn't the default colour be black (or the blue we originally used)?

Reply:
The color specified here _is_ black (technically speaking it's no color, but it produces black since Red, Green, and Blue are not introduced).

If you'd like the default to be the original blue, I can change this to (0,0,102,0) (this would effectively reproduce our original call of `glColor4f(0.0f, 0.0f, 0.4f, 0.0f)`)

Reply:
But, alpha is `0` here. How is this working?

Reply:
Had to look this one up lol. It works because normal windows that are non-transparent do not carry the alpha value to the framebuffer, so the `0` for alpha is just ignored - we never had to worry before so 0 worked for us. However, when transparency is enabled, it will _not_ be ignored.

I'll set alpha to 1 by default.